src/sys/signalfd.rs

```diff
@@ -148,14 +148,17 @@ mod tests {
     fn create_signalfd() {
         let mask = SigSet::empty();
         let fd = SignalFd::new(&mask);
-        assert!(fd.is_ok());
+        fd.expect("assert failed");
```

Comment:
"assert failed" isn't a helpful error message.  Could you replace them with something more useful?  Or if not, then just use `.unwrap()`.

Reply:
https://github.com/nix-rust/nix/pull/1786

The idea was to be able to tell that this is an assert instead of a random unwrap, but I don't mind either way.